runtime/src/prioritization_fee_cache.rs

```diff
@@ -208,7 +205,7 @@ impl PrioritizationFeeCache {
         }
     }
 
-    /// Update with a list of non-vote transactions' tx_priority_details and tx_account_locks; Only
+    /// Update with a list of non-vote transactions' tx_compute_budget_details and tx_account_locks; Only
     /// transactions have both valid priority_detail and account_locks will be used to update
```

Comment:
nit: `priority_detail` => `compute_budget_details` (also, i'd prefer removing the seemingly-unneeded `tx_` prefixes.

Reply:
38ecc8b9e2c1ed85067cc45a620795a7c481f98e